### PR TITLE
I solved problems in third lab

### PR DESCRIPTION
**Pull request by iceandrise**
You wrote me some problems of my lab in this repo: https://github.com/iceandrise/itirod2/pull/2
But I decide to push my changes in this repo
Waiting for your mark or comments.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,54 +11,49 @@
...
       </div>
-
-      <div class="auth_buttons">
-        <a href="logout.html">
+      <div class = "wrapper__auth">
+        <a class = "auth_buttons" href="logout.html">
           <button class="button">Log Out</button>
         </a>
-        <a href="index.html">
+        <a class = "auth_buttons" href="index.html">
```
Comment (katetushkan) [inline index.html]: not solved.

File: index.html [hunk 2]
```diff
@@ -11,54 +11,49 @@
...
+    <div class="main__wrapper"> 
+      <div class="counter__container"> 
+        <div class="counter"> 
+          <h1 class="counter__title">Counter</h1> 
           <div class="wrapper">
-            <div class="wrapper__output">
-              <label class="wrapper__output_postfix">Completed tasks:</label>
-              <div class="cut"></div>
+            <div class="wrapper__counter">
+              <span class="wrapper__counter_completed">Completed tasks:</span>
```
Comment (katetushkan) [inline index.html]: not solved. should be `<p>`

File: index.html [hunk 3]
```diff
@@ -11,54 +11,49 @@
...
+      <div class="filter__container">
         <div class="filter">
           <h1 class="filter__title">Search</h1>
           <div class="wrapper">
             <div class="wrapper__filter">
               <input type="text" class="input" name="input__filter" />
               <button class="button">Search</button>
             </div>
-            <h3 class="tagfilter__title">Tag filter</h3>
+            <h1 class="tagfilter__title">Tag filter</h1>
```
Comment (katetushkan) [inline index.html]: not solved. https://github.com/iceandrise/itirod2/pull/2#discussion_r841498032
It's related to everything.

File: index.html [hunk 4]
```diff
@@ -98,13 +93,13 @@ <h3 class="tagfilter__title">Tag filter</h3>
...
 
-      <div class="container">
+      <div class="collection__container">
         <div class="collection">
           <h1 class="collect__title">Collection</h1>
           <div class="wrapper">
             <div class="wrapper__collect">
-              <label class="wrapper__output_postfix">Count of your pets:</label>
-              <div class="cut"></div>
+              <span class="wrapper__output_count">Count of your pets:</span>
```
Comment (katetushkan) [inline index.html]: should be `<p>`

File: logout.html [hunk 5]
```diff
@@ -8,28 +8,23 @@
...
-           <a href="index.html">
-             <span class="bounty">BountyDo</span>
-           </a>   
+             <small class="bounty">BountyDo</small>  
           </div>  
        
-          <div class="auth_buttons">
-            <a href="signin.html">
+          <div class = "wrapper__auth">
+            <a  class = "auth_buttons" href="signin.html">
```
Comment (katetushkan) [inline logout.html]: same here https://github.com/iceandrise/itirod2/pull/2#discussion_r841495151

File: signin.css [hunk 6]
```diff
@@ -6,26 +6,22 @@
...
 
-  .header {
-    display: inline-flex;
+  header {
     justify-content: space-between;
     width: 100%;    
-    height: 120px;    
     margin-bottom: 3px;
-    background-color:rgb(48, 139, 48);
+    background-color:forestgreen;
```
Comment (katetushkan) [inline signin.css]: when I told css variables I meant [this](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).

File: signup.css [hunk 7]
```diff
@@ -104,18 +74,44 @@
...
+  .wrapper__signup{
+    display: flex;
+      align-items:flex-start;
+      justify-content:center;
+      width: 100%;
+      margin-top: 50px;
+  
+  }
+  
+     .main {
```
Comment (katetushkan) [inline signup.css]: fix tabs

File: signup.css [hunk 8]
```diff
@@ -127,7 +123,7 @@
   cursor: pointer;
   }
 
-  input[type=text], input[type=username] { 
+  .main input[type=text], input[type=username] {
```
Comment (katetushkan) [inline signup.css]: use class names

**Review by katetushkan — CHANGES_REQUESTED** [review]
deploy link?

File: logout.html (hunk 5 above)
Comment (iceandrise) [reply logout.html]: Sorry, but I don't understand my mistake... Can you explain more?

**Comment by iceandrise** [comment]
@katetushkan  could you review please?

File: README.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
+*Курилович Карина 953504*
```
Comment (katetushkan) [inline README.md:2]: Screenshot 2022-04-21 at 21 22 12
It still doesn't have a mobile version.

**Review by katetushkan — CHANGES_REQUESTED** [review]

**Comment by katetushkan** [comment]
image

Still no mobile version

**Review by katetushkan — CHANGES_REQUESTED** [review]
image

Still no mobile version

**Comment by iceandrise** [comment]
@katetushkan sorry, I updated deploy [link](https://6277b5c40f4e573efb2aa87c--classy-lolly-eece77.netlify.app/), please check

**Comment by iceandrise** [comment]
@katetushkan also signin and signup pages will be remove at lab 5.

**Review by katetushkan — APPROVED** [review]
Okay. Then it's a <kbd>8</kbd>.

**Pull request merged**